docs/Splashkit/Tutorials/Reviews/Review On Mouse Inputs Functionality Guide.md

```diff
@@ -0,0 +1,33 @@
+# Title : Tutorial Review On Mouse Inputs Functionality Guide
+
+
+Date Reviewed: 28 August 2024
+
+Reviewed by: Mounika Angadipeta
+
+## Description:
+A tutorial review of the tutorial has been done. The code in this tutorial has been run to ensure that it works and outputs the results. The code has all the three languages C++, C#, Python. Modified part of the text description of the sentence and Mouse location Functionality code in C# to make the content clearer.
+
+## Confirmation of Passing Quality Checks:
+
+- [x] The tutorial is free of spelling and grammatical errors.
+- [x] The tutorial is easy to follow.
+- [x] The tutorial has:
+  - [x] C# using top level statements
+  - [x] C++
+  - [x] Python
+- [x] The code passes all the Quality Checks.
+
+## Links:
+
+Website Link: https://splashkit.io/guides/input/1-mouse-button-inputs/
+
+Pull Request: https://github.com/thoth-tech/splashkit.io-starlight/pull/161
+
+## Improvements and suggestions:
+
+If you have any suggestions for improvements or changes, add here:
+
+- updated the sentence formation to make it meaningfull
+- Fixed the Code error of Mouse Location Functionality Mouse x in C#.
```

Comment:
I would move this up into the description, as this section is more so for "future" improvements.